app/models/reporting/patient_states_per_month.rb

```diff
@@ -0,0 +1,6 @@
+module Reporting
```

Comment:
Its unfortunate we have another module named `Reporting` now when we already have `Reports`.  Could we just call this `Reports` to be consistent?  I think `set_table_name` will do the right thing for us...we can then use whatever module we want.

Reply:
I was thinking about the github/scientist branching when I created a different namespace. I thought we could eventually just split across 2 different namespaces for the comparison. Also, "reporting" followed naturally from the table name prefixes.

There's also the existing matviews, and I didn't want to conflate them with the new tables. How would you feel about merging these two namespaces after the github/scientist experiment? We'll have a good sense of the final picture by then.

Reply:
We'll rename the namespace to `ReportingPipeline` to make the intention very clear. Also created a [story](https://app.clubhouse.io/simpledotorg/story/3900/consolidate-reporting-model-namespaces) to merge the namespaces.